python/cuml/test/test_trustworthiness.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2018-2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytest
+from sklearn.manifold.t_sne import trustworthiness as sklearn_trustworthiness
```

Comment:
Pytests are required for end-to-end but we need some C++ unit tests as well to verify that the score is returning the expected output. When given a small pre-defined set of inputs, a C++ test should verify that the output matches what you would expect. Without C++ unit tests, it becomes very hard to troubleshoot these algorithms when regressions and bugs appear. 

Once this PR merged, I hope to use it for testing UMAP (and eventually t-SNE) in the C++ layer.